src/appShell/App/PortalFooter.tsx

```diff
@@ -156,8 +156,7 @@ export default class PortalFooter extends React.Component<
                                     </If>
                                     <If
                                         condition={
-                                            AppConfig.serverConfig.app_name ===
-                                            'public-portal'
+                                            AppConfig.serverConfig.show_twitter
```

Comment:
maybe `skin.right_nav.show_twitter`? See: https://docs.cbioportal.org/2.3-customization/portal.properties-reference#hide-sections-in-the-right-navigation-bar